code/datums/elements/loot_drop.dm

```diff
@@ -0,0 +1,33 @@
+/**
+ * # Loot on Death element
+ *
+ * A mob with this element will create the provided atoms when it dies.
+ */
+/datum/element/loot_drop
```

Comment:
I was wondering why you included a new element because I swore I knew of bileworms dropping loot, and then it hit me: Why don't we just extend `/datum/element/death_drops`? Seems kinda silly to have two elements that do the exact same thing...